swig/yui.i

```diff
@@ -214,12 +216,14 @@ class Exception;
 %include yui/YImage.h
 %include yui/YInputField.h
 %include yui/YIntField.h
+%include yui/YItemSelector.h
 %include yui/YLabel.h
 %include yui/YLayoutBox.h
 %include yui/YLogView.h
 %include yui/YMacro.h
 %include yui/YMacroPlayer.h
 %include yui/YMacroRecorder.h
+%include yui/YMenuBar.h
```

Comment:
YMenuWidget is missing, please add also
%include yui/YMenuWidget.h

Reply:
It doesn't support indirect includes? Eeek...

Adding.